File: src/Entity/Activite.php
<?php

namespace App\Entity;

use App\Repository\ActiviteRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Entity(repositoryClass=ActiviteRepository::class)
 */
class Activite
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank
     */
    private $nom_activite;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank
     */
    private $type_activite;

    /**
     * @ORM\Column(type="integer")
     * @Assert\PositiveOrZero
     * @Assert\NotBlank
     */
    private $prix_activite;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNomActivite(): ?string
    {
        return $this->nom_activite;
    }

    public function setNomActivite(string $nom_activite): self
    {
        $this->nom_activite = $nom_activite;

        return $this;
    }

    public function getTypeActivite(): ?string
    {
        return $this->type_activite;
    }

    public function setTypeActivite(string $type_activite): self
    {
        $this->type_activite = $type_activite;

        return $this;
    }

    public function getPrixActivite(): ?int
    {
        return $this->prix_activite;
    }

    public function setPrixActivite(int $prix_activite): self
    {
        $this->prix_activite = $prix_activite;

        return $this;
    }
    public function __toString()
    {

        return $this->nom_activite;
    }
}

File: Tunidiscovery-Mobile/src/com/mycompany/myapp/entities/Activite.java

package com.mycompany.myapp.entities;

/**
 *
 * @author IDIZUOB
 */
public class Activite {
    private int id;
    private String nom_activite;
    private String type_activite;
    private int prix_activite;

    public Activite() {
    }

    
    public Activite(String nom_activite, String type_activite, int prix_activite) {
        this.nom_activite = nom_activite;
        this.type_activite = type_activite;
        this.prix_activite = prix_activite;
    }
    

    public Activite(int id, String nom_activite, String type_activite, int prix_activite) {
        this.id = id;
        this.nom_activite = nom_activite;
        this.type_activite = type_activite;
        this.prix_activite = prix_activite;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom_activite() {
        return nom_activite;
    }

    public void setNom_activite(String nom_activite) {
        this.nom_activite = nom_activite;
    }

    public String getType_activite() {
        return type_activite;
    }

    public void setType_activite(String type_activite) {
        this.type_activite = type_activite;
    }

    public int getPrix_activite() {
        return prix_activite;
    }

    public void setPrix_activite(int prix_activite) {
        this.prix_activite = prix_activite;
    }

    @Override
    public String toString() {
        return "Activite{" + "id=" + id + ", nom_activite=" + nom_activite + ", type_activite=" + type_activite + ", prix_activite=" + prix_activite + '}';
    }
    
    
    
    
}
